src/edge/edgeManager.ts

```diff
@@ -325,6 +326,10 @@ export class EdgeManager {
             case Constants.LANGUAGE_CSHARP:
                 launchFile = Constants.launchCSharp;
                 mapObj.set(Constants.appFolder, "/app");
+                const csprojPath: string = path.join(slnPath, Constants.moduleFolder, moduleName, moduleName + Constants.csharpProjectFlieExtensionName);
+                const csprojStr: string = await fse.readFile(csprojPath, "utf-8");
+                const targetFramework: string = csprojStr.match(/<TargetFramework>(.+?)<\/TargetFramework>/)[1];

```

Comment:
Is it possible there is no targetFramework?

Reply:
targetFramework must be existing, otherwise, it would be an invalid csproj file. 
And this file is come from template and controlled by ourselves, so we can safely consider it must be existing

Reply:
need trim spaces?

Reply:
updated